app/mailers/commontator/subscriptions_mailer.rb

```diff
@@ -0,0 +1,50 @@
+class Commontator::SubscriptionsMailer < ActionMailer::Base
+  helper Commontator::SharedHelper
+  def comment_created(comment, recipients)
+    setup_variables(comment, recipients)
+
+    mail(@mail_params).tap do |message|
+      message.mailgun_recipient_variables = @mailgun_recipient_variables if @using_mailgun
+    end
+  end
+
+  protected
+
+  def setup_variables(comment, recipients)
+    @comment = comment
+    commentUser = User.find(comment.creator_id).first_name + " " + User.find(comment.creator_id).last_name
+
+    @comment_header = commentUser + " has commented on #{@comment.thread.commontable.name}"
```

Comment:
we can use full interpolation here
```@comment_header = "#{commentUser} has commented on #{@comment.thread.commontable.name}"```

Reply:
using interpolation for both comment and reply lines here.  Also changed local var name from currentUser to current_user.... :)

Reply:
![image](https://user-images.githubusercontent.com/60527788/80843420-1ffb6080-8bb9-11ea-8517-1b90ee64c76e.png)